### ISSUE TITLE
Suggestions for new layouts

### ISSUE THREAD

**Issue by IjzerenHein** [IjzerenHein/famous-flex#50]
Post any suggestions for new layouts here.

**Comment by IjzerenHein** [comment]
Spherical layout +1 (lays out renderables in a 3 dimensional sphere)

**Comment by grimen** [comment]
Haha, you are on fire - great work! :) Tweakable swipe menus? I.e. triggered from edges/corners of containers.

**Comment by IjzerenHein** [comment]
Nice!

Sounds more like a widget, but it definitely needs a layout as well. If you have some examples/screenshots/animated gifs please share them. Perhaps we should create a separate thread dedicated for discussing this widget.

**Comment by grimen** [comment]
Well I basically been doing something like this:

From official Famo.us University tutorials:
http://famo.us/university/lessons/#/famous-102/timbre/36

Inofficial library I found but clashed with RenderController and FamousFlex somehow so could not get it to behave other than in isolated test:
https://github.com/markmarijnissen/famous-sidepanel

**Comment by ArEnSc** [comment]
I just want to say this is amazing work I am not sure why famo.us isn't working on widgets like these.

**Comment by matt-d-rat** [comment]
Masonry layout (similar to this: http://masonry.desandro.com), which is like a grid, but supports widgets which can be of variable height and perhaps variable width.

**Comment by bardu** [comment]
+1 for masonry

**Comment by IjzerenHein** [comment]
Sweet! Never new such a layout was called masonry.

+1 for masonry

**Comment by grimen** [comment]
+1

With respect to custom widths too is a quite common scenario: http://erikjohanssonphoto.com/work/imagecats/personal

**Comment by matt-d-rat** [comment]
Agreed, it would need to support variable height and widths for each of the items in the grid.

**Comment by IjzerenHein** [comment]
Good suggestions. Let's keep the content in this thread purely for voting for (new) layouts and discuss specific layouts in dedicated threads. I've created an thread for the masonry layout: #57
